plugins/s3_auth/s3_auth.cc

```diff
@@ -711,6 +746,7 @@ TSRemapDoRemap(void *ih, TSHttpTxn txnp, TSRemapRequestInfo * /* rri */)
 
   if (s3) {
     TSAssert(s3->valid());
+    s3->acquire(); // Increasement ref-count
```

Comment:
Hmmm, I think the 60s shield you from any potential race condition here, since this is part of DoRemap. Regardless, that "race" applies to any configuration that's stored as part of the remap config, right ?